ci/provisioning/post_provision_config_nodes_EL_7.sh

```diff
@@ -38,6 +38,7 @@ group_repo_post() {
 }
 
 distro_custom() {
+
```

Comment:
Whitespace changes are OK if they improve readability or coding standards, but if they're the only change in the file then they shouldn't be in this PR.